### ISSUE TITLE
2. button aria-label

### ISSUE THREAD

**Issue by neener** [Simone-Subal-Gallery/simone-subal#7]
![Image](https://user-images.githubusercontent.com/6674492/221698947-2ed34a42-97e6-45fd-b91b-a9d6e4d5f677.png)

Buttons should include text explaining their functionality, and if icons are used as buttons, a screen- reader only text or an "aria-label" attribute should be used for that description.

1hr
